chart/env/prod.yaml

```
@@ -341,7 +341,7 @@ search:
   uvicornNumWorkers: "9"
   nodeSelector:
     role-datasets-server-search: "true"
-  replicas: 8
+  replicas: 16
```

We have only 6 machines that can have up to 2 pods each. You can set it to 12 maximum
```suggestion
  replicas: 12
```

see https://github.com/huggingface/infra/blob/69465295f1331c68dd7768aafa3a497f0ab6044e/projects/hub/00-aws/variables/hub-aws-prod-us-east-1.tfvars#L190-L199

---

we can also increase the value in infra

---

I set it to 12 until we increase the value in infra.